frontends/web/src/routes/device/bitbox02/setup/password.tsx

```diff
@@ -15,17 +15,30 @@
  */
 
 import { useTranslation } from 'react-i18next';
+import { i18n } from '../../../../i18n/i18n';
 import { View, ViewContent, ViewHeader } from '../../../../components/view/view';
 import { Backup } from '../../../../api/backup';
 import { PasswordEntry } from '../components/password-entry/password-entry';
 import { Status } from '../../../../components/status/status';
 import { MultilineMarkup } from '../../../../utils/markup';
 import { convertDateToLocaleString } from '../../../../utils/date';
+import { A } from '../../../../components/anchor/anchor';
+import { Info } from '../../../../components/icon';
+import style from './password.module.css';
 
 type Props = {
   errorText: string | undefined;
 }
 
+const getSupportLink = () => {
+  switch (i18n.resolvedLanguage) {
+  case 'de':
+    return 'https://shiftcrypto.support/help/de-de/3-basics/84-wie-lange-sollte-mein-geratepasswort-sein';
+  default:
+    return 'https://shiftcrypto.support/help/en-us/3-basics/84-wie-lange-sollte-mein-geratepasswort-sein';
```

Comment:
nit: if you change from german to english the uri still contains the german verision. (silly knowledge base tool).

please use 
https://shiftcrypto.support/help/en-us/3-basics/84-how-long-should-my-device-password-be